presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1360,10 +1355,15 @@ private void executeRenameTasksForAbort()
         private void deleteEmptyStagingDirectories(List<DeclaredIntentionToWrite> declaredIntentionsToWrite)
```

Comment:
I don't know if they should be put inside this method or not. Should temp table be considered as staging directories?

Reply:
Good point. Extracted into a separate `deleteTemporaryTableDirectories` method